### PR TITLE
Load the AST250 ROM into address 0x0 in QEMU

### PR DESCRIPTION
**Pull request by rjoleary**
Signed-off-by: Ryan O'Leary <ryanoleary@google.com>

### CODE REVIEW

File: src/mainboard/ast/ast25x0/Makefile.toml [hunk 1]
```diff
@@ -27,10 +28,5 @@ args = ["xbuild", "@@split(CARGO_ARGS, )"]
...
 [tasks.run]
 dependencies = ["default"]
-command = "qemu-system-arm"
-args = ["-machine", "virt", "-bios", "${TARGET_DIR}/oreboot.bin", "-nographic", "-S", "-s", ]
-
-[tasks.2500]
-dependencies = ["default"]
-command = "qemu-system-arm"
-args = ["-m", "512m", "-machine", "ast2500-evb", "-bios", "${TARGET_DIR}/oreboot.bin", "-nographic", ]
+command = "/home/ryanoleary/repos/openbmc/qemu/build-arm/arm-softmmu/qemu-system-arm"
```
Comment (rminnich) [inline src/mainboard/ast/ast25x0/Makefile.toml]: this one won't end well, I'm not you :-)

**Comment by rminnich** [comment]
oh hey! could you put your qemu config file in src/mainboard/ast/ast2500 as part of this CL along with the git ref of qemu that you built?
config-qemu-abcdefg form?

File: src/mainboard/ast/ast25x0/Makefile.toml (hunk 1 above)
Comment (rjoleary) [reply src/mainboard/ast/ast25x0/Makefile.toml]: Whooops. I guess the important thing is to use openbmc qemu -- they've emulated more of the soc.

Comment (rjoleary) [reply src/mainboard/ast/ast25x0/Makefile.toml]: Done!

**Pull request merged**